### DISCUSSION TITLE
Option to pin favorite skin in skin list

### DISCUSSION THREAD

**Discussion by Pwnisher** [ppy/osu#23534] [Ideas] [answered]
Would love to see this feature since I change skin frequently, and one of those skins is in the near bottom of the list therefore the need to scroll down.

Here a concept I have in mind:
![Pin favorite skin](https://github.com/ppy/osu/assets/22348598/f83504b6-b07b-4414-b260-a9449189ffd6)
The left version is where the pinned skins are placed under the official skins (argon, triangles, classic).
The right version is where the pinned skins are placed on both top of the list and above the official skins.

**Comment by peppy** [comment, marked as answer]
We do have plans on improving the whole skin selection flow, but I do agree that having the ability to pin skins is something we want at the end.

**Comment by ShikkesoraSIM** [comment]
My implementation here! #37649.
